packages/shared/src/svg/LogoRecruiterSvg.tsx

```diff
@@ -0,0 +1,66 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+
+export type LogoRecruiterProps = {
+  className?: {
+    container?: string;
+  };
+};
+
+export const LogoRecruiterSvg = ({
+  className,
+}: LogoRecruiterProps): ReactElement => {
+  return (
+    <svg
```

Comment:
Can't we just put it on cloudinary maybe?

Reply:
for logo I think its betters its svg so it loads instant no?